src/components/Footer/index.js

```diff
@@ -47,46 +27,29 @@ const useStyles = makeStyles(theme => ({
     [theme.breakpoints.up('md')]: {
       justifyContent: 'flex-end'
     }
-  },
-  community: {
-    width: '50%',
-    [theme.breakpoints.up('sm')]: {
-      width: 'auto'
-    }
-  },
-  project: {
-    width: '50%',
-    [theme.breakpoints.up('sm')]: {
-      width: 'auto'
-    }
   }
 }));
 
 function Footer(props) {
   const classes = useStyles(props);
 
   return (
-    <Grid container className={classes.root}>
-      <Grid
-        item
-        xs={12}
-        container
-        alignItems="flex-start"
-        className={classes.layout}
-      >
-        <Grid item xs={12} sm={7} className={classes.about}>
-          <About />
-        </Grid>
-        <Grid item xs={12} sm={5} container className={classes.organisation}>
-          <Grid item className={classes.community}>
+    <div className={classes.root}>
+      <Container>
+        <Grid item xs={12} container spacing={3} alignItems="flex-start">
```

Comment:
Not sure about mixing `Container` and `Grid item ... ` though... Wouldn't `Grid container` (without the `item`) work? I'm under the impression `Grid item ... .` should always be a child in `Grid container`...

Reply:
Good catch. Unintentional to have `item` and `container` in one. Otherwise there's a difference between `<Container/>` and `<Grid container/>`. Detailed here - https://spectrum.chat/material-ui/help/grid-vs-box-and-now-vs-container~73cef09f-1eb9-4d0f-a3a3-d46c44232524?m=MTU1MTc0MDk5NTI4NA==